x-pack/plugins/enterprise_search/server/utils/create_ml_inference_pipeline.ts

```diff
@@ -41,14 +42,14 @@ export const createAndReferenceMlInferencePipeline = async (
   pipelineName: string,
   modelId: string,
   sourceField: string,
-  destinationField: string,
+  destinationField: string | null | undefined,
```

Comment:
When do we expect to get `null` vs `undefined`?

Reply:
Hm, good question. I was thinking the API that could be called with `"destination_field": null` (e.g. from Postman), but from the Kibana UI I agree it's unnecessary.